examples/fargate-serverless/main.tf

```diff
@@ -252,31 +177,12 @@ module "vpc" {
   default_security_group_tags   = { Name = "${local.name}-default" }
 
   public_subnet_tags = {
-    "kubernetes.io/cluster/${local.name}" = "shared"
```

Comment:
no longer required with recent versions of ALB controller

Reply:
interesting. where is this documented? 
we are on the latest release of AWS LB Controller, and still tag out SGs

Reply:
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/docs/deploy/subnet_discovery.md#common-tag

Reply:
Security group tags is separate, and that is still required. This is just subnet tags

Reply:
right right right!